models/ios/acls/ios_acls.yaml

```diff
@@ -0,0 +1,679 @@
+---
+ANSIBLE_METADATA: |
+    {
+      'metadata_version': '1.1',
+      'status': ['preview'],
+      'supported_by': 'network'
+    }
+
+DOCUMENTATION: |
+  module: ios_acls
+  version_added: 2.10
+  short_description: Configure and manage Named or Numbered ACLs on IOS devices.
+  description: This module configures and manages the Named or Numbered on IOS platforms.
+  author: Sumit Jaiswal (@justjais)
+  notes:
+  - Tested against Cisco IOSv Version 15.2 on VIRL
+  - This module works with connection C(network_cli).
+    See L(IOS Platform Options,../network/user_guide/platform_ios.html).
+  options:
+    config:
+      description: A dictionary of ACL options
+      type: list
+      elements: dict
+      suboptions:
+        name:
+          description: The name of the ACL
+          type: str
+          suboptions:
+            standard:
+              description: Standard Access List
+              type: bool
+            extended:
+              description: Extended Access List
+              type: bool
+            resequence:
+              description: Resequence Access List
+              suboptions:
+                start_sequence:
+                  description:
+                  - Starting Sequence Number.
+                  - Refer to vendor documentation for valid values.
+                  type: int
+                increment_sequence:
+                  description:
+                  - Step to increment the sequence number.
+                  - Refer to vendor documentation for valid values.
+                  type: int
+        number:
+          description:
+          - IP ACLs number.
+          - Refer to vendor documentation for valid values.
+          type: int
+        entries:
+          description: The entries within the ACL
+          mutually_exclusive: [[name], [number]]
+          elements: dict
+          required_together: [[action, protocol, source, destination]]
+          type: list
+          suboptions:
+            action:
+              description: Specify the action
+              type: str
+              choice: ['permit', 'deny']
+            source:
```

Comment:
This option should also allow the `any` ability which should be mutually exclusive with `address` and `wildcard_bits`.